octokit/search.go

```diff
@@ -2,47 +2,60 @@ package octokit
 
 import (
 	"github.com/jingweno/go-sawyer/hypermedia"
-	"net/url"
 )
 
-var SearchURL = Hyperlink("search{/type}?q={query}{&page,per_page,sort,order}")
+var SearchURITemplate = "search{/type}?q={query}{&page,per_page,sort,order}"
```

Comment:
Done, though I don't clearly understand why having having `Hyperlink` instead of `string` would be useful.